components/furnituremix/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "devDependencies": {
+        "@tailwindcss/line-clamp": "^0.4.0",
```

Comment:
Should we move this to the package.json that's in the application folder?

Reply:
totally! Good catch. I am doing it